### DISCUSSION TITLE
People app no longer supported?

### DISCUSSION THREAD

**Discussion by BlazorDashboard** [microsoft/WindowsAppSDK#1741] [Q&A] [answered]
I have noticed People app can no longer be launched directly from Windows 11 but only from the Mail app. 

There is also an Open issue at the following for quite some time:

https://github.com/microsoft/WindowsAppSDK/issues/1157

My question is whether the ContactPicker API for People app still be supported in Windows App SDK? Should new Windows App SDK app be using this at all? Will be grateful for any advice.

**Comment by andrewleader** [comment]
Hi @BlazorDashboard, thanks for asking, I've reached out to some teams to find you an answer, we should have an answer in a day or two. Thanks!

**Comment by WebAssemblyMan** [comment]
Hi @andrewleader is there an update on this issue with regard to the People app or ContactPicker? Thanks.

**Comment by andrewleader** [comment, marked as answer]
Hi, sorry for the lack of update. We don't have anyone actively working on this space. If you've noticed a concrete issue with ContactPicker not working, please let us know. The removal of the user-facing People app is a separate user-facing topic and not a developer topic, and you can report that feedback through Feedback Hub.
